app/assets/stylesheets/components/_card.scss

```diff
@@ -62,9 +62,10 @@ be) overwritten.
 .tournaments, .my-tournaments {
   background-image: url(space-background.jpg);
   background-repeat: repeat;
-  height: auto;
+  background-size: 1000px;
```

Comment:
Won't this leave a border if someone has a wide screen? is there no option like 100% or something, or using object fit cover

Reply:
Try using in scss
min-width: 100vh;